timescaledb-restore/docker-compose.yaml

```
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  timescaledb:
+    platform: linux/amd64  
+    image: timescale-restore:1
```

```suggestion
    image: timescale-restore:1
    build: .
```
This allows you to skip the `docker build` step.